third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -1426,6 +1426,33 @@ impl SignatureToken {
     pub fn num_nodes(&self) -> usize {
         self.preorder_traversal().count()
     }
+
+    pub fn instantiate(&self, subst_mapping: &BTreeMap<u16, SignatureToken>) -> SignatureToken {
```

Comment:
Not asking for any changes but just curious, is there a specific reason why `BTreeMap` is used here? In some other contexts, `&[SignatureToken]` might also be a common  (or even more performant) representation